docs/Annotate-MAF-File.md

```diff
@@ -0,0 +1,31 @@
+# Annotate a MAF File 📄
+
+MAF files can be annotated using either the web interface or the command line client. The [Mutation Mapper](https://www.cbioportal.org/mutation_mapper) is a web-based tool and provides an easy-to-use graphical user interface for annotating MAF files. On the other hand, the command line client offers a more flexible approach for users who prefer command-line interfaces. Both options allow you to add additional information to the MAF file, which can be useful for downstream analysis and interpretation.
+
+
+## Annotate a MAF file using the command line client
+
+To annotate a specific MAF file using the command line client, we recommend using the [genome-nexus-annotation-pipeline](https://github.com/genome-nexus/genome-nexus-annotation-pipeline). This powerful tool adds annotations to the MAF files quickly and easily, with options for customizing the annotations to suit your specific needs.
```

Comment:
```suggestion
To annotate a specific MAF file using the command line client, we recommend using the [genome-nexus-annotation-pipeline](https://github.com/genome-nexus/genome-nexus-annotation-pipeline). This tool adds annotations to the MAF files, with options for customizing the annotations to suit your specific needs.
```
Thought it's maybe good to have a less "advertising" tone here for the technical docs